### PR TITLE
chore: add getAlgoOrderDetails method

### PR DESCRIPTION
**Pull request by SebastianBoehler**
## Summary
Added the getAlgoOrderDetails method since it was missing

### CODE REVIEW

File: src/types/rest/request/trade.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export interface AlgoLongHistoryRequest {
   limit?: string;
 }
 
-export interface AlgoOrderRequest {
+export interface PlaceAlgoOrderRequest {
```
Comment (tiagosiebler) [inline src/types/rest/request/trade.ts:36]: Renaming this is a breaking change (for anyone already importing & using `AlgoOrderRequest`), it's also now the only "request" interface prefixed with "Place". Please revert this. The rest looks good to me, thanks for the PR!

Comment (SebastianBoehler) [reply src/types/rest/request/trade.ts]: You're right, reverted :)

**Comment by SebastianBoehler** [comment]
Just saw the issue https://github.com/tiagosiebler/okx-api/issues/40 and included it as well here

File: src/types/rest/request/trade.ts (hunk 1 above)
Comment (tiagosiebler) [reply src/types/rest/request/trade.ts]: Thanks! The rest looks good, if you're finished, please also increase the version number in the package.json file - it'll make the release process easier :)

Comment (SebastianBoehler) [reply src/types/rest/request/trade.ts]: Done ;) thanks for creating the package, am using it almost daily. Happy to contribute.
Ready for merge 🥳

Comment (tiagosiebler) [reply src/types/rest/request/trade.ts]: Glad it's helping! And thanks again for the PR. Will be on npm shortly.

**Comment by tiagosiebler** [comment]
Had to push a fix, build was failing due to a bad import - slipped past. Released via #42

**Pull request merged**